tests/UI/commonTests/BO/createDeleteCustomer.js

```diff
@@ -0,0 +1,78 @@
+require('module-alias/register');
+// Import utils
+const helper = require('@utils/helpers');
+
+// Import login test
+const loginCommon = require('@commonTests/loginBO');
+
+// Import BO pages
+const dashboardPage = require('@pages/BO/dashboard');
+const customersPage = require('@pages/BO/customers');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+// Import expect from chai
+const {expect} = require('chai');
+
+let browserContext;
+let page;
+
+/**
+ * Function to delete customer
+ * @param customerData {CustomerData} Data to set to delete customer
+ * @param baseContext {string} String to identify the test
+ */
+function deleteCustomer(customerData, baseContext = 'commonTests-deleteCustomer') {
```

Comment:
I think the function name could be mistaken with the function on the customer page.
Can we have something more specific that means it's a Common test?

Reply:
Yes, it's true!
WDYT of **deleteCustomerTest** ?